diff -uNr gphoto2-2.4.14.orig/configure.ac gphoto2-2.4.14/configure.ac
--- gphoto2-2.4.14.orig/configure.ac	2012-04-15 18:05:54.000000000 +0200
+++ gphoto2-2.4.14/configure.ac	2012-06-25 17:25:10.283187148 +0200
@@ -5,7 +5,7 @@
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_MACRO_DIR([auto-m4])
 AC_CONFIG_AUX_DIR([auto-aux])
-AM_INIT_AUTOMAKE([-Wall -Werror gnu 1.9 dist-bzip2 check-news])
+AM_INIT_AUTOMAKE([-Wall gnu 1.9 dist-bzip2 check-news])
 
 if test ! -d "$srcdir/m4m"; then
 AC_MSG_ERROR([
