--- gphoto2-2.3.0/m4m/gp-check-library.m4~	2006-11-30 03:20:02.000000000 +0200
+++ gphoto2-2.3.0/m4m/gp-check-library.m4	2006-12-09 17:50:05.318105081 +0200
@@ -171,7 +171,7 @@
 				_[$1]_REQ_3="regexp([$3], [\(>=\|>\|<\|<=\|=\)[ \t]*\([0-9]+\)\.\([0-9]+\)\.\([0-9]+\).*],   [\4])"
 				_[$1]_REQ_4="regexp([$3], [\(>=\|>\|<\|<=\|=\)[ \t]*\([0-9]+\)\.\([0-9]+\)\.\([0-9]+\)(.*)], [\5])"
 				# split installed version number via shell and sed
-				_[$1]_VERSION="$("${[$1][_CONFIG_PROG]}" --version)"
+				_[$1]_VERSION="$("${[$1][_CONFIG_PROG]}" --version | awk '{print $NF}')"
 				_[$1]_VER_1="$(echo "${_[$1]_VERSION}" | sed 's/\([[0-9]]*\).*/\1/g')"
 				_[$1]_VER_2="$(echo "${_[$1]_VERSION}" | sed 's/\([[0-9]]*\)\.\([[0-9]]*\).*/\2/g')"
 				_[$1]_VER_3="$(echo "${_[$1]_VERSION}" | sed 's/\([[0-9]]*\)\.\([[0-9]]*\)\.\([[0-9]]*\).*/\3/g')"
--- gphoto2-2.3.0/configure.ac~	2006-12-09 17:51:56.420609247 +0200
+++ gphoto2-2.3.0/configure.ac	2006-12-09 17:58:55.890063006 +0200
@@ -79,7 +79,7 @@
 dnl ---------------------------------------------------------------------------
 dnl Check for libgphoto2
 dnl ---------------------------------------------------------------------------
-GP_CHECK_LIBRARY([LIBGPHOTO2], [libgphoto2], [>= 2.2.1.4])
+GP_CHECK_LIBRARY([LIBGPHOTO2], [libgphoto2], [>= 2.3.0])
 
 
 AC_CHECK_FUNCS([strptime])
