diff -urN gphoto2-2.0.orig/configure.in gphoto2-2.0/configure.in
--- gphoto2-2.0.orig/configure.in	Mon Feb 25 14:30:20 2002
+++ gphoto2-2.0/configure.in	Sun Apr  7 12:55:19 2002
@@ -225,10 +225,10 @@
 	cdk_prefix="$withval")
 if $try_cdk; then
 	CPPFLAGS_save="$CPPFLAGS"
-	CPPFLAGS="-I$cdk_prefix/include -I$cdk_prefix/include/cdk $CPPFLAGS"
+	CPPFLAGS="-I$cdk_prefix/include -I$cdk_prefix/include/cdk -I/usr/include/ncurses $CPPFLAGS"
 	AC_CHECK_HEADER(cdk.h, [
 		have_cdk=true
-		CDK_CFLAGS="-I$cdk_prefix/include -I$cdk_prefix/include/cdk"
+		CDK_CFLAGS="-I$cdk_prefix/include -I$cdk_prefix/include/cdk -I/usr/include/ncurses"
 		CDK_LIBS="-L$cdk_prefix/lib -lcdk -lncurses"
 		AC_DEFINE(HAVE_CDK)
 		LDFLAGS_save="$LDFLAGS"
