diff -urN gphoto2-2.0.org/configure.in gphoto2-2.0/configure.in
--- gphoto2-2.0.org/configure.in	Tue Apr 30 17:25:11 2002
+++ gphoto2-2.0/configure.in	Tue Apr 30 19:23:28 2002
@@ -387,6 +387,10 @@
 AC_SUBST(AA_LIBS)
 AC_SUBST(AA_CFLAGS)
 
+AM_CONDITIONAL(ENABLE_GTK_DOC, false)
+AM_CONDITIONAL(ENABLE_FIGURES, false)
+AM_CONDITIONAL(ENABLE_RPM, false)
+
 dnl ---------------------------------------------------------------------------
 dnl Configure libgphoto2-port
 dnl ---------------------------------------------------------------------------
--- gphoto2-2.1.0/libgphoto2_port/configure.in.orig	Tue Jun 25 14:52:16 2002
+++ gphoto2-2.1.0/libgphoto2_port/configure.in	Thu Aug 29 20:25:49 2002
@@ -88,7 +88,7 @@
                         have_ltdl=true])])
 fi
 if test x$have_ltdl != "xtrue"; then
-        AC_CHECK_FUNC(dlopen,,[
+        AC_CHECK_FUNC(dlopen, [true], [
                 AC_CHECK_LIB(dl, dlopen,[LTDL_LIBS="-ldl"],[AC_ERROR([
 *** Could not determine how to handle
 *** shared libraries!])])])
@@ -147,7 +147,7 @@
 try_baudboy_msg=true
 try_ttylock=true
 try_lockdev=true
-AC_ARG_WITH(serial, [  --without-serial        Don't compile in SERIAL support],
+AC_ARG_WITH(serial, [  --without-serial        Don't compile in SERIAL support],[
 	if test x$withval = xno; then
 		have_serial=false
 		serial_msg=no
@@ -155,7 +155,7 @@
 		try_ttylock=false
 		try_lockdev=false
 	fi
-)
+])
 if $have_serial; then
 	IOLIB_SUBDIRS="$IOLIB_SUBDIRS serial"
 fi
@@ -166,21 +166,21 @@
 dnl  (2) ttylock.h
 dnl  (3) lockdev.h
 dnl -----------------------------------------
-AC_ARG_WITH(baudboy, [  --without-baudboy       Don't use baudboy library],
+AC_ARG_WITH(baudboy, [  --without-baudboy       Don't use baudboy library],[
 	if test x$withval = xno; then
 		try_baudboy=false
 	fi
-)
-AC_ARG_WITH(ttylock, [  --without-ttylock       Don't use ttylock library],
+])
+AC_ARG_WITH(ttylock, [  --without-ttylock       Don't use ttylock library],[
 	if test x$withval = xno; then
 		try_ttylock=false
 	fi
-)
-AC_ARG_WITH(lockdev, [  --without-lockdev       Don't use lockdev library],
+])
+AC_ARG_WITH(lockdev, [  --without-lockdev       Don't use lockdev library],[
 	if test x$withval = xno; then
 		try_lockdev=false
 	fi
-)
+])
 if $try_baudboy; then
 	AC_CHECK_HEADER(baudboy.h,
 		baudboy_msg=yes
@@ -190,15 +190,15 @@
 	)
 fi
 if $try_ttylock; then
-	AC_CHECK_HEADER(ttylock.h,
+	AC_CHECK_HEADER(ttylock.h,[
 		AC_MSG_CHECKING(if ttylock needs lockdev)
-		AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],
+		AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],[
 			AC_MSG_RESULT(no)
 			AC_DEFINE(HAVE_TTYLOCK,,[Define if you have ttylock based tty locking.])
 			ttylock_msg=yes
 			try_lockdev=false,
 			LDFLAGS="$LDFLAGS -llockdev"
-			AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],
+			AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],[
 				AC_MSG_RESULT(yes)
 				AC_DEFINE(HAVE_TTYLOCK,,[Define if you have ttylock based tty locking.])
 				ttylock_msg=yes
@@ -206,10 +206,10 @@
 				SERIAL_LDFLAGS=-llockdev,
 				AC_MSG_RESULT(unknown),
 				AC_MSG_RESULT(unknown)
-			),
+			]),
 			AC_MSG_RESULT(unknown)
-		)
-	)
+		])
+	])
 fi
 if $try_lockdev; then
 	AC_CHECK_LIB(lockdev, dev_lock,[
@@ -229,28 +229,28 @@
 dnl ******
 usb_msg="no (required for USB cameras, http://sourceforge.net/projects/libusb/)"
 try_usb=true
-AC_ARG_WITH(usb, [  --without-usb           Don't compile in USB support],
+AC_ARG_WITH(usb, [  --without-usb           Don't compile in USB support],[
 	if test x$withval = xno; then
 		try_usb=false
 	fi
-)
+])
 if $try_usb; then
 	AC_PATH_PROG(LIBUSB_CONFIG,libusb-config)
 	if test -n "${LIBUSB_CONFIG}"; then
 		CPPFLAGS_save="$CPPFLAGS"
 		CPPFLAGS="$CPPFLAGS `libusb-config --cflags`"
-		AC_CHECK_HEADER(usb.h,
+		AC_CHECK_HEADER(usb.h,[
 			usb_msg="yes"
 			IOLIB_SUBDIRS="$IOLIB_SUBDIRS usb"
 			USB_LIBS="`libusb-config --libs`"
 			USB_CFLAGS="`libusb-config --cflags`"
-                        LIBUSB_VER="`libusb-config --version`")
+                        LIBUSB_VER="`libusb-config --version`"])
 		CPPFLAGS="$CPPFLAGS_save"
 	fi
 
 	# This 'test' call uses non-standard syntax and will fail on most
 	# systems, but it's just informational so any warning can be ignored.
-	if test "$LIBUSB_VER" \< "0.1.5"; then
+	if (echo "$LIBUSB_VER" | grep -E "^0\.[01]\.[01234]"); then
 		AC_MSG_WARN([
 *** You need at least version 0.1.5 of the libusb library for USB support
 *** http://sourceforge.net/projects/libusb/
@@ -281,8 +281,8 @@
 dnl ********
 dnl gtk-doc?
 dnl ********
-GP_CHECK_DOC_DIR()dnl
-GP_BUILD_DOCS()dnl
+GP_CHECK_DOC_DIR
+GP_BUILD_DOCS
 
 AC_OUTPUT([
 libgphoto2_port.spec
--- gphoto2-2.1.0/packaging/rpm/Makefile.am~	Thu Aug 29 23:06:37 2002
+++ gphoto2-2.1.0/packaging/rpm/Makefile.am	Thu Aug 29 23:06:37 2002
@@ -6,24 +6,3 @@
 
 EXTRA_DIST = package.spec.in
 
-clean-generic:
-	rm -rf BUILD RPMS SPECS SOURCES SRPMS
-
-rpm: Makefile package.spec
-if RPM_LIBUSB_DEVEL
-	@echo "Building gphoto2 RPM with USB support and dependency on the libusb RPM package."
-else
-	@echo "#############################################################################"
-	@echo "# Building gphoto2 RPM packages without USB support.                        #"
-	@echo "# Install both the libusb and the libusb-devel RPM package for USB support. #"
-	@echo "#############################################################################"
-	@sleep 3
-endif
-	$(MAKE) clean
-	mkdir -p BUILD SPECS SOURCES SRPMS
-	for arch in i386 i486 i586 i686 alpha noarch; do \
-		mkdir -p "RPMS/$${arch}"; \
-	done
-	ln -f "$(top_srcdir)/@PACKAGE@-@VERSION@.tar.gz" SOURCES/
-	ln -f package.spec SPECS/@PACKAGE@-@VERSION@.spec
-	rpm --define "_topdir $$(pwd)" -ba SPECS/@PACKAGE@-@VERSION@.spec
--- gphoto2-2.1.0/doc/Makefile.am~	Thu Aug 29 23:36:50 2002
+++ gphoto2-2.1.0/doc/Makefile.am	Thu Aug 29 23:36:50 2002
@@ -10,15 +10,8 @@
 #
 
 # distribute manual Portable Document Format (PDF) if possible
-manual_pdf = @manual_pdf@
 
-# distribute manual in PostScript (PS) format if possible
-manual_ps = @manual_ps@
-
-# build HTML manual if possible
-manual_html = @manual_html@
-
-CLEAN_FILES = $(manual_pdf) $(manual_ps) figures-stamp
+CLEAN_FILES = figures-stamp
 
 EXTRA_DIST = DAEMON build_OS2.txt 					\
 	camlib_devel.lyx 						\
@@ -26,18 +19,14 @@
 	camlib_devel.sgml camlib_devel.txt camlib_devel-y.eps 		\
 	frontend_devel.lyx frontend_devel.sgml frontend_devel.txt 	\
 	gphoto2-cli.txt gphoto2.txt					\
-	gphoto2.xml $(man_MANS) $(manual_pdf) $(manual_ps)		\
+	gphoto2.xml $(man_MANS) 					\
 	FAQ
 
 man_MANS = gphoto2.1 gphoto2.3 gphoto2_port.3
 
 docdir = @DOC_DIR@
 
-doc_DATA = gphoto2-cli.txt gphoto2.txt FAQ $(manual_pdf) $(manual_ps)
-
-figures-stamp:
-	$(MAKE) -C figures
-	touch $@
+doc_DATA = gphoto2-cli.txt gphoto2.txt FAQ
 
 ########################################################################
 # The following rules are for regenerating man pages, pdf, html and ps
@@ -50,30 +39,10 @@
 
 
 ########################################################################
-if XMLTOPDF
-gphoto2.pdf: gphoto2.xml figures-stamp
-	test -e $$(basename $<) || ln -s $< # symlink file from srcdir to builddir if required
-	$(XMLTO) pdf $$(basename $<) # work on local file or symlink
-endif
-########################################################################
-
-
-########################################################################
-if XMLTOPS
-# doesn't work if builddir != srcdir
-gphoto2.ps: gphoto2.xml figures-stamp
-	test -e $$(basename $<) || ln -s $< # symlink file from srcdir to builddir if required
-	$(XMLTO) ps $$(basename $<) # work on local file or symlink
-endif
-########################################################################
-
-
-########################################################################
-if XMLTOMAN
 # works if builddir != srcdir
 $(man_MANS): gphoto2.xml
 	$(XMLTO) man $<
-endif
+
 ########################################################################
 
 
@@ -83,39 +52,29 @@
 
 
 ########################################################################
-if XMLTOHTML
-# doesn't work correctly if builddir != srcdir
-
 # do not empty this variable
-htmldir=@HTML_DIR@
+htmldir=$(datadir)/doc/gphoto2
 
 TARGET_DIR=$(htmldir)/manual
 
 manual: gphoto2.xml
 	echo "$$(pwd) $@ $<"
 	$(XMLTO) html -o "$@" "$<"
-	cd $@ && rm -f figures && ln -s ../figures figures # to view docs in src tree
-	touch "$@"
 
 clean_local_manual = clean-local-manual
 clean-local-manual:
 	rm -rf "manual/"
 
 install_data_local_manual = install-data-local-manual
-install-data-local-manual: manual figures-stamp
+install-data-local-manual: manual
 	$(INSTALL) -d -m 0755 $(DESTDIR)$(TARGET_DIR)
 	$(INSTALL_DATA) ./manual/*.html $(DESTDIR)$(TARGET_DIR)
-	$(INSTALL) -d -m 0755 $(DESTDIR)$(TARGET_DIR)/figures
-	$(INSTALL_DATA) ./figures/*.png $(DESTDIR)$(TARGET_DIR)/figures
 
 # copy HTML manual to distribution
 dist_hook_manual = dist-hook-manual
 dist-hook-manual: manual figures-stamp
 	tar cf - manual/*.html | (cd $(distdir); tar xvf -)
-if ENABLE_FIGURES
 	tar cf - figures/*.png | (cd $(distdir)/manual/; tar xvf -)
-endif
-endif
 ########################################################################
 
 
@@ -130,5 +89,5 @@
 dist-hook: $(dist_hook_manual)
 	@echo "Dummy $@ target"
 
-all-local: $(man_MANS) $(manual_pdf) $(manual_ps) $(manual_html)
+all-local: $(man_MANS)
 	@echo "Dummy $@ target"
--- gphoto2-2.1.0/doc/figures/Makefile.am~	Fri Aug 30 01:36:17 2002
+++ gphoto2-2.1.0/doc/figures/Makefile.am	Fri Aug 30 01:36:17 2002
@@ -4,7 +4,7 @@
 
 EXTRA_DIST = architecture.fig libgphoto2-api.fig libgphoto2-camlib.fig libgphoto2_port.fig gphotofs.fig
 
-figuredir = @HTML_DIR@/figures
+figuredir = htmldir=$(datadir)/doc/gphoto2/figures
 
 if ENABLE_FIGURES
 
