diff -urN gphoto2-2.0.org/configure.in gphoto2-2.0/configure.in
--- gphoto2-2.0.org/configure.in	Tue Apr 30 17:25:11 2002
+++ gphoto2-2.0/configure.in	Tue Apr 30 19:23:28 2002
@@ -387,6 +387,10 @@
 AC_SUBST(AA_LIBS)
 AC_SUBST(AA_CFLAGS)
 
+AM_CONDITIONAL(ENABLE_GTK_DOC, false)
+AM_CONDITIONAL(ENABLE_FIGURES, false)
+AM_CONDITIONAL(ENABLE_RPM, false)
+
 dnl ---------------------------------------------------------------------------
 dnl Configure libgphoto2-port
 dnl ---------------------------------------------------------------------------
diff -urN gphoto2-2.0.org/libgphoto2_port/configure.in gphoto2-2.0/libgphoto2_port/configure.in
--- gphoto2-2.0.org/libgphoto2_port/configure.in	Tue Apr 30 17:25:11 2002
+++ gphoto2-2.0/libgphoto2_port/configure.in	Tue Apr 30 19:06:53 2002
@@ -60,13 +60,13 @@
 dnl         )
 dnl )
 if $no_libtool; then
-	AC_CHECK_FUNC(dlopen,,
+	AC_CHECK_FUNC(dlopen,[true],[
                 AC_CHECK_LIB(dl, dlopen,
                         LTDL_LIBS="-ldl",
                         AC_ERROR([
 *** Could not determine how to handle
 *** shared libraries!
-])))
+]))])
 fi
 AC_SUBST(LTDL_LIBS)
 
@@ -119,7 +119,7 @@
 try_baudboy_msg=true
 try_ttylock=true
 try_lockdev=true
-AC_ARG_WITH(serial, [  --without-serial        Don't compile in SERIAL support],
+AC_ARG_WITH(serial, [  --without-serial        Don't compile in SERIAL support],[
 	if test x$withval = xno; then
 		have_serial=false
 		serial_msg=no
@@ -127,7 +127,7 @@
 		try_ttylock=false
 		try_lockdev=false
 	fi
-)
+])
 if $have_serial; then
 	IOLIB_SUBDIRS="$IOLIB_SUBDIRS serial"
 fi
@@ -138,39 +138,39 @@
 dnl  (2) ttylock.h
 dnl  (3) lockdev.h
 dnl -----------------------------------------
-AC_ARG_WITH(baudboy, [  --without-baudboy       Don't use baudboy library],
+AC_ARG_WITH(baudboy, [  --without-baudboy       Don't use baudboy library],[
 	if test x$withval = xno; then
 		try_baudboy=false
 	fi
-)
-AC_ARG_WITH(ttylock, [  --without-ttylock       Don't use ttylock library],
+])
+AC_ARG_WITH(ttylock, [  --without-ttylock       Don't use ttylock library],[
 	if test x$withval = xno; then
 		try_ttylock=false
 	fi
-)
-AC_ARG_WITH(lockdev, [  --without-lockdev       Don't use lockdev library],
+])
+AC_ARG_WITH(lockdev, [  --without-lockdev       Don't use lockdev library],[
 	if test x$withval = xno; then
 		try_lockdev=false
 	fi
-)
+])
 if $try_baudboy; then
-	AC_CHECK_HEADER(baudboy.h,
+	AC_CHECK_HEADER(baudboy.h,[
 		baudboy_msg=yes
 		try_ttylock=false
 		try_lockdev=false
 		AC_DEFINE(HAVE_BAUDBOY)
-	)
+	])
 fi
 if $try_ttylock; then
-	AC_CHECK_HEADER(ttylock.h,
+	AC_CHECK_HEADER(ttylock.h,[
 		AC_MSG_CHECKING(if ttylock needs lockdev)
-		AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],
+		AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],[
 			AC_MSG_RESULT(no)
 			AC_DEFINE(HAVE_TTYLOCK)
 			ttylock_msg=yes
 			try_lockdev=false,
 			LDFLAGS="$LDFLAGS -llockdev"
-			AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],
+			AC_TRY_LINK([#include <ttylock.h>],[ttylocked ("/dev/foo");],[
 				AC_MSG_RESULT(yes)
 				AC_DEFINE(HAVE_TTYLOCK)
 				ttylock_msg=yes
@@ -178,19 +178,19 @@
 				SERIAL_LDFLAGS=-llockdev,
 				AC_MSG_RESULT(unknown),
 				AC_MSG_RESULT(unknown)
-			),
+			]),
 			AC_MSG_RESULT(unknown)
-		)
-	)
+		])
+	])
 fi
 if $try_lockdev; then
-	AC_CHECK_LIB(lockdev, dev_lock,
-		AC_CHECK_HEADER(lockdev.h,
+	AC_CHECK_LIB(lockdev, dev_lock,[
+		AC_CHECK_HEADER(lockdev.h,[
 			AC_DEFINE(HAVE_LOCKDEV)
 			lockdev_msg=yes
 			SERIAL_LDFLAGS=-llockdev
-		)
-	)
+		])
+	])
 fi
 AC_SUBST(SERIAL_LDFLAGS)
 
@@ -199,28 +199,28 @@
 dnl ******
 usb_msg=no
 try_usb=true
-AC_ARG_WITH(usb, [  --without-usb           Don't compile in USB support],
+AC_ARG_WITH(usb, [  --without-usb           Don't compile in USB support],[
 	if test x$withval = xno; then
 		try_usb=false
 	fi
-)
+])
 if $try_usb; then
 	AC_PATH_PROG(LIBUSB_CONFIG,libusb-config)
 	if test -n "${LIBUSB_CONFIG}"; then
 		CPPFLAGS_save="$CPPFLAGS"
 		CPPFLAGS="$CPPFLAGS `libusb-config --cflags`"
-		AC_CHECK_HEADER(usb.h,
+		AC_CHECK_HEADER(usb.h,[
 			usb_msg=yes
 			IOLIB_SUBDIRS="$IOLIB_SUBDIRS usb"
 			USB_LIBS="`libusb-config --libs`"
 			USB_CFLAGS="`libusb-config --cflags`"
-                        LIBUSB_VER="`libusb-config --version`")
+                        LIBUSB_VER="`libusb-config --version`"])
 		CPPFLAGS="$CPPFLAGS_save"
 	fi
 
 	# This 'test' call uses non-standard syntax and will fail on most
 	# systems, but it's just informational so any warning can be ignored.
-	if test "$LIBUSB_VER" \< "0.1.5"; then
+	if (echo "$LIBUSB_VER" | grep -E "^0\.[01]\.[01234]"); then
 		AC_MSG_WARN([
 *** You need at least version 0.1.5 of the libusb library for USB support
 *** http://libusb.sourceforge.net/
@@ -251,8 +251,8 @@
 dnl ********
 dnl gtk-doc?
 dnl ********
-GP_CHECK_DOC_DIR()dnl
-GP_BUILD_DOCS()dnl
+GP_CHECK_DOC_DIR
+GP_BUILD_DOCS
 
 AC_OUTPUT([
 libgphoto2_port.spec
