--- gphoto2-2.1.4/configure.in~	Mon Jul 26 10:45:54 2004
+++ gphoto2-2.1.4/configure.in	Mon Jul 26 10:45:54 2004
@@ -16,7 +16,7 @@
 dnl ---------------------------------------------------------------------------
 dnl i18n support
 dnl ---------------------------------------------------------------------------
-ALL_LINGUAS="az cs da de en_GB es fi fr hu it ja nl no pt_BR ro ru sk sl sr sv uk zh_CN"
+ALL_LINGUAS="az cs da de en_GB es fi fr hu it ja nl nb pt_BR ro ru sk sl sr sv uk zh_CN"
 # EVIL HACKS
 AC_MSG_CHECKING([for po/Makevars requiring evil hack])
 if test -f po/Makevars.template
--- gphoto2-2.1.4/configure~	Mon Jul 26 10:45:50 2004
+++ gphoto2-2.1.4/configure	Mon Jul 26 10:45:50 2004
@@ -3070,7 +3070,7 @@
 _ACEOF
 
 
-ALL_LINGUAS="az cs da de en_GB es fi fr hu it ja nl no pt_BR ro ru sk sl sr sv uk zh_CN"
+ALL_LINGUAS="az cs da de en_GB es fi fr hu it ja nl nb pt_BR ro ru sk sl sr sv uk zh_CN"
 # EVIL HACKS
 echo "$as_me:$LINENO: checking for po/Makevars requiring evil hack" >&5
 echo $ECHO_N "checking for po/Makevars requiring evil hack... $ECHO_C" >&6
